Resources/Prototypes/Datasets/Names/books.yml

```
@@ -0,0 +1,262 @@
+  - codex
+  - memories
+  - compendium
+
+- type: dataset
+  id: book_genre
+  values:
+  - work of crime fiction
+  - comedy
+  - horror story
+  - poem
+  - novella
+  - chronicle
+  - work of science-fiction
+  - fantasy story
+  - romance
+  - thriller
+  - work of historical fiction
+  - biography
+  - adventure story
+  - drama
+
+- type: dataset
+  id: book_hint_appearance
+  values:
+  - ancient
+  - shabby
+  - dirty
+  - unusual
+  - faded
+  - nasty
+  - dusty
+  - scary
+  - bloody
+  - bright
+  - dubious
+  - intriguing
+  - ugly
+  - crooked
+  - crumpled
+  - dirty
+  - elegant
+  - ornate
+  - weathered
+  - chrisp
+  - lavish
+  - tattered
+  - polished
+  - embossed
+  - mismatched
+  - gilded
+  - strange
+
+- type: dataset
+  id: book_character
+  values:
+  - clown
+  - mime
+  - reporter
+  - butcher
+  - bartender
+  - janitor
+  - engineer
+  - scientist
+  - guard
+  - doctor
+  - chemist
+  - prisoner
+  - researcher
+  - trader
+  - captain
+  - lizard
+  - moth
+  - diona
+  - cat-girl
+  - cat
+  - corgi
+  - dog
+  - opossum
+  - sloth
+  - syndicate agent
+  - revenant
+  - rat king
+  - ninja
+  - space dragon
+  - revolutionary
+  - nuclear operative
+  - narsie cultist
+  - ratwar cultist
+  - greytider
+  - arachnid
+  - vox
+  - dwarf
+
+- type: dataset
+  id: book_character_trait
+  values:
+  - stupid
+  - smart
+  - funny
+  - attractive  
+  - charming
+  - nasty
+  - dying
+  - old
+  - young
+  - rich
+  - poor
+  - popular
+  - absent-minded
+  - stern
+  - сharismatic
+  - stoic
+  - cute
+  - dwarven
+  - beer-smelling 
+  - joyful
+  - painfully beautiful
+  - robotic
+  - holographic
+  - hysterically laughing
+
+- type: dataset
+  id: book_event
+  values:
+  - a zombie outbreak
+  - a nuclear  explosion
+  - a mass murder
+  - a sudden depressurization
+  - a blackout
+  - the starvation of the main characters
+  - a wasting illness
+  - love at first sight
+  - a rush of inspiration
+  - the occurrence of some mystical phenomena
+  - divine intervention
+  - the characters' own selfish motives
+  - an unforeseen deception
+  - the resurrection of one of these characters from the dead
+  - the terrible torture of the protagonist
+  - the inadvertent loosing of a gravitational singularity
+  - a psychic prediction of future events
+  - an antimatter explosion
+  - a chance meeting with a cat-girl
+  - drinking far too much alcohol
+  - eating way too much pizza
+  - having a quarrel with a close friend
+  - the sudden loss of their home in a fiery blaze
+  - the loss of a PDA
+
+- type: dataset
+  id: book_action
+  values:
+  - share in a kiss with a
+  - strangle to death a
+  - manage to blow apart a
+  - manage to win a game of chess against a
+  - narrowly lose a game of chess against a
+  - reveal the hidden secrets of a
+  - manipulate a
+  - sacrifice upon an altar a
+  - attend the wedding of a
+  - join forces to defeat their common enemy, a
+  - are forced to work together to escape a
```

```suggestion
  - resort to working together to escape a
```
Because of a preceding adverb, passive voice looks funny here